html-css/override-extend-variables/step1-instructions.md

```diff
@@ -1,11 +1,11 @@
-## Add code to the workspace
-
-Before we get started, let's add some code to the workspace that we can modify.
-
+## Add code to the workspace
+ 
+Before you get started, let's add some code to the workspace that you can modify.
```

Comment:
The step markdowns shouldn't be using "you" or "we" anymore @gheake2

Reply:
Before you get started, let's add some code to the workspace that you can modify.
sb
Add code to the workspace that can be modified.